### PR TITLE
CAPA v29.0.0: Update `observability-bundle`.

### PR DESCRIPTION
**Pull request by marieroque**
This PR updates the `observability-bundle` to `v1.5.2` on the CAPA v29.0.0 release.
Previous release deployed the `observability-bundle@1.3.4`.
Here the CHANGELOG: https://github.com/giantswarm/observability-bundle/blob/main/CHANGELOG.md

Main changes are:
- kube-prometheus-stack updated with an upgrade of prometheus-operator CRDs
- `alloy` app added to the bundle
- fix CNP issues
- add `observability-policies` app to deploy Observability Kyverno Policies https://github.com/giantswarm/giantswarm/issues/31155

### Checklist
- [x] Roadmap issue created
- [x] Release uses latest stable Flatcar
- [x] Release uses latest Kubernetes patch version

### Triggering e2e tests

To trigger the E2E test for each new Release added in this PR add a comment with the following:

`/run releases-test-suites`

If you want to trigger conformance tests you can do so by adding a comment similar to the following:

`/run conformance-tests RELEASE_VERSION=v29.0.0 PROVIDER=capa`

For more details see the [README.md](/README.md#running-tests-against-prs)

### CODE REVIEW

**Review by Gacko — CHANGES_REQUESTED** [review]
There's an issue with a GitHub action atm that would forge a release once your PR is getting merged. I therefore block merging by requesting changes, just to be sure it's not getting merged accidentally before that issue has been fixed.

In the meantime we can run the releases test suite:

/run releases-test-suites

**Comment by AverageMarcus** [comment]
The issue with getting the wrong Release should (hopefully) now be fixed...

/run releases-test-suites

**Comment by Gacko** [comment]
I'm still working on fixing the automation, @marieroque. It should get merged today, then I can rebase and merge your PR also.

File: capa/v29.0.0/release.yaml
Comment (Gacko) [inline capa/v29.0.0/release.yaml:114]: Franco changed this to `kyverno-crds` in their PR: https://github.com/giantswarm/releases/pull/1319. I'll try to get their PR merged, so I can rebase yours afterwards.

**Comment by Gacko** [comment]
@marieroque: I added the changelog from `observability-bundle`. Apart from that, I didn't change anything, so the passed tests should still be valid and the PR safe to skip CI.

Are my changes fine with you?

**Review by Gacko — APPROVED** [review]

**Comment by marieroque** [comment]
> @marieroque: I added the changelog from `observability-bundle`. Apart from that, I didn't change anything, so the passed tests should still be valid and the PR safe to skip CI.
> 
> Are my changes fine with you?

Maybe we should add something related to the new default app `observability-policies` in the README

**Comment by Gacko** [comment]
Sure, done! 🙂

**Comment by Gacko** [comment]
I removed `observability-policies` for now for following reasons:

Even though we implemented it in the `cluster` chart, the `cluster-aws` chart does not rely on that `cluster` chart version, yet. Also, even if `cluster-aws` chart would rely on a `cluster` chart version that included the `observability-policies` app, it would not yet enable it via the corresponding provider integration values.

The same basically applies to the so called `giantswarm-cluster-suite` app. We have implemented it in the `cluster` chart, but not yet enabled it in the `cluster-aws` chart and therefore can not have it in the release for now.

To have it included in the release, we first need to release a new version of the `cluster` chart including that app and then include this new `cluster` chart version in the `cluster-aws` chart, enabling it there and afterwards release a new version of the `cluster-aws` chart.

cc @nprokopic

**Review by Gacko — APPROVED** [review]

**Pull request merged**
